### DISCUSSION TITLE
sample generative model

### DISCUSSION THREAD

**Discussion by martinjrobins** [pints-team/pints#1681] [Ideas/Science]
each pints log-likelihood defines a generative model, which you can sample (e.g. for simulating fake data), it would be nice if there was a method to sample from each log-likelihood, given a parameter set

**Comment by ben18785** [comment]
Hmm, interesting. Would it be an issue though that a log-likelihood contains data, so a user might think it's somehow sampling from a sorta posterior distribution? (Whereas I guess it would just be sampling data from a distribution with the parameters supplied by the user?)

**Comment by HOLL95** [comment]
@martinjrobins would you mind clarifying what you meant by "sample from each log-likelihood" and how it would differ from e.g. ```problem.evaluate(random_values)```?
